tests/test_reporting.py

```diff
@@ -132,6 +134,60 @@ def test_protected_results(self):
         assert response.status_code == 401
         assert not response.json
 
+    def test_adherence_sort(self):
+        from portal.models.adherence_data import sort_by_visit_key
+        sort_me = {
+            "Month 3": {
+                "qb": "CRV_recurring_3mo_period v2",
+                "site": "CRV",
+                "visit": "Month 3",
+                "status": "Completed",
+                "consent": "11 - Mar - 2023 07: 42: 46 ",
+                "country ": None,
+                "user_id": 4,
+                "site_code": "",
+                "entry_method": "interview_assisted",
+                "completion_date ": "19 - Jun - 2023 07: 42:46 ",
+                "oow_completion_date": ""
+            },
+            "Month 12": {
+                "qb": "CRV Baseline v2",
+                "site": "CRV",
+                "visit": "Month 12",
+                "status": "Overdue",
+                "consent": "20 - May - 2023 07: 42:46 ",
+                "country ": None,
+                "user_id ": 3,
+                "study_id": "study user 3",
```

Comment:
Are these normally numeric, and this is just fake data?

Reply:
just fake test data - typically the `study_id` is ###-##.